src/ViewModeResolver.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Drupal\format_strawberryfield;
+
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Cache\UseCacheBackendTrait;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\strawberryfield\StrawberryfieldUtilityServiceInterface;
+
+/**
+ * View Mode resolver service.
+ *
+ * @ingroup format_strawberryfield
+ */
+class ViewModeResolver implements ViewModeResolverInterface {
+  use UseCacheBackendTrait;
```

Comment:
let's add a an extra line here before the trait , just for readability (sorry, I'm lacking coffee) 👴

Reply:
Yeah, I had my coffee(s). Not even worth the change. Dismiss!